proto/metapb.proto

```diff
@@ -61,6 +61,42 @@ message RegionEpoch {
     uint64 version = 2;
 }
 
+message BucketStats {
+    // total read (point get) in bytes of each bucket
+    repeated uint64 read_bytes = 1;
+
+    // total scan in bytes of each bucket 
+    repeated uint64 scan_bytes = 2;
+
+    // total write in bytes of each bucket
+    repeated uint64 write_bytes = 3;
+
+    // total read (point get) qps of each bucket
+    repeated uint64 read_qps = 4;
+
+    // total scan qps of each bucket
+    repeated uint64 scan_qps = 5;
```

Comment:
Is it necessary to distinguish between point and scan？I think the first version can be simpler.

Reply:
ok